implementations/elixir/ockam/ockam_hub/test/hub/service/alias_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule Test.Hub.Service.AliasTest do
+  use ExUnit.Case
+
+  alias Ockam.Hub.Service.Alias, as: AliasService
+  alias Ockam.Hub.Service.Echo, as: EchoService
+  alias Ockam.Router
+  alias Test.Utils
+
+  test "alias test" do
+    {:ok, _worker, worker_address} =
+      Test.Hub.Service.AliasTestWorker.start_link(address: "bdf373h28asdf")
+
+    {:ok, _alias, _alias_address} = AliasService.start_link(address: "cdf373h48asdf")
+    {:ok, _echo, _echo_address} = EchoService.start_link(address: "ddf373h47asdf")
```

Comment:
i don't remember, but does the test terminate linked processes? Maybe we need an explicit cleanup?

Reply:
As the process is linked to the test process, it should be terminated.